src/web/src/asset_DELETE.ecpp

```diff
@@ -56,14 +57,16 @@ bool database_ready;
     std::string checked_id; // user-friendly identifier
 
     // sanity check
+    std::string id = request.getArg ("id");
     {
```

Comment:
Why this brace (and scoping) move? I see we log `id` below, but could actually log the `checked_id` instead as it is the value we use and trust (or bail out in this clause). The idea for `checked*` stuff was that the original untrusted input variables do not leak around (checked ones might be sanitized, etc.).

Reply:
Also it seems that `id` is logged in many other files, where this apparently does not cause scoping issues... maybe it should have ;)

Reply:
Scope is still local file, just a bit extended to be able to log it. While I agree we can use this variable in the block, and then use checked one, the result should still be the same, right?

Reply:
Should be... but only as long as the `checked_*` values we use are a guaranteed copy, not a conversion product, of the input `*` query arguments.